pootle/apps/pootle_app/migrations/0001_initial.py

```diff
@@ -28,6 +28,12 @@ class Migration(migrations.Migration):
             },
             bases=(models.Model, pootle.core.mixins.treeitem.CachedTreeItem),
         ),
+        migrations.RunSQL("ALTER TABLE `pootle_app_directory` ROW_FORMAT=DYNAMIC"),
+        migrations.AlterField(
+            model_name='directory',
```

Comment:
I looked at (pootle/apps/pootle_store/migrations/0002_make_suggestion_user_not_null.py)[https://github.com/Yelp/pootle/blob/master/pootle/apps/pootle_store/migrations/0002_make_suggestion_user_not_null.py], which uses AlterField and it wasn't case sensitive, so I followed that. I'll confirm